Makefile

```diff
@@ -103,7 +103,7 @@ editorconfig: editorconfig-checker
 .PHONY: test
 test:
 	@echo "test => ./..."
-	@go test -v ./...
+	@go test -v -race ./...
```

Comment:
note: this was to enhance the concurrency CEL test (we should've added before anyways)

Reply:
[nit] Now that you're changing this, what about this?
```make
TEST_OPTS ?= -v -race
.PHONY: test
test:
	@echo "test => ./..."
	@go test $(TEST_OPTS) ./...
```
This would allow users to use custom options like `-count`, that are usually handy during development, while keeping the desired defaults.

Reply:
i don't want to enable -race by default especially for the e2e tests which makes it unnecessary & slow that's why i added -race only here but open to suggestions

Reply:
but yeah will do that